README.md

```diff
@@ -58,7 +61,45 @@ deployment options.
 }
 ```
 
-### Adding new integrations
+#### Local execution using Docker
+You can also execute the agent building and running a Docker image:
+```shell
+docker build -t local_agent --target generic --platform=linux/amd64 .;\
+docker run --rm --name local_agent -p8081:8081 -ePORT=8081 -it local_agent
+```
+Or running the latest dev image from DockerHub:
+```shell
+docker run --rm --name dev_agent -p8081:8081 -ePORT=8081 -it montecarlodata/pre-release-agent:latest-generic
```

Comment:
For future devs, can we include a line explaining how to run the unit tests with the image? Thanks!